node/src/command.rs

```diff
@@ -296,10 +310,20 @@ pub fn run() -> Result<()> {
 		}
 		None => {
 			let runner = cli.create_runner(&cli.run.normalize())?;
+			let chain_spec = &runner.config().chain_spec;
+			let is_pint_dev = chain_spec.is_dev();
+
+			set_default_ss58_version(chain_spec);
 
 			runner.run_node_until_exit(|config| async move {
 				let para_id = chain_spec::Extensions::try_get(&*config.chain_spec).map(|e| e.para_id);
 
+				if is_pint_dev {
```

Comment:
In this checking

Reply:
should we `self.id() == "pint_dev"` instead?